src/Symfony/Component/Serializer/Mapping/Factory/CacheClassMetadataFactory.php

```diff
@@ -32,6 +32,11 @@ class CacheClassMetadataFactory implements ClassMetadataFactoryInterface
      */
     private $cacheItemPool;
 
+    /**
+     * @var array
```

Comment:
I'd suggest initializing the property and removing the annotation.

Reply:
done, thanks for the review